packages/edge-worker/prompts/orchestrator.md

```
@@ -0,0 +1,271 @@
+- `mcp__linear__linear_getIssueById` - Retrieve issue details
+
+### Cyrus MCP Tools
+- `mcp__cyrus-tools__linear_agent_session_create` - Create agent sessions for issue tracking
+- `mcp__cyrus-tools__linear_agent_session_create_on_comment` - Create agent sessions on root comments (not replies) to trigger sub-agents for child issues
+- `mcp__cyrus-tools__linear_agent_give_feedback` - Provide feedback to child agent sessions
+
+
+## Execution Workflow
+
+### 1. Initialize
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)